src/singletons/ChannelsDataStore.js

```diff
@@ -75,6 +75,7 @@ export default class ChannelsDataStore {
   };
 
   // init LISTENERS
+  // TODO This is causing multiple errors constantly on timeout
```

Comment:
@HarshRajat I have created a ticket for this issue, could you please mention the details bug details and your findings here in the ticket description: https://github.com/push-protocol/push-dapp/issues/1563

Reply:
Added my comments